docusaurus/docs/iOS/guides/push-notifications.md

```diff
@@ -1,3 +1,121 @@
 ---
 title: Push Notifications
----
\ No newline at end of file
+---
+
+First step on a way to setting up push notifications is authentication.
+Stream supports both **Certificate-based provider connection trust (.p12 certificate)** and **Token-based provider connection trust (JWT)**. Token based authentication is the preferred way to setup push notifications. This method is easy to setup and provides strong security. You can find more on setting up authentication [here](https://getstream.io/chat/docs/php/push_ios/?language=swift).
+
+:::caution
+Make sure you've taken care of authentication before proceeding to the next steps
+:::
+
+## Testing Push Notifications setup
+
+Once authentication is taken care of, you can test your setup as explained below.
+
+First of all make sure you have at least one [device associated](#managing-devices-for-testing-purposes).
```

Comment:
It's interesting to have something required below this. Perhaps we should move the "Managing Devices" sections above and move this down, so the guide logically follows?

Reply:
Makes sense, reworked this a bit to be more consecutive